### ISSUE TITLE
Rebuilt the apk,can't show anything.

### ISSUE THREAD

**Issue by canwushuang** [updeshxp/scrcpy-android#4]
I use the new server.jar(scrcpy version 1.13) in assets to rebuild apk, it can't show anything.But change the old one and rebuild, it's ok.

**Comment by updeshxp** [comment]
Yup, the new server requires some structural changes which is what I'm working on.

**Comment by canwushuang** [comment]
Thanks a lot! It need some to change in "org/las2mile/scrcpy" folder. It will take you much time on it.

**Comment by updeshxp** [comment]
Thie author has released a new version:
https://gitlab.com/las2mile/scrcpy-android
Could you Please check on the same if this issue is present.

**Comment by sd3970566** [comment]
如何实现在客户端进行截图保存 png   到/sdcard/test.png      服务端需要作出什么改动吗？
